src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/gui/action/CorrelationJMXFile.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jmeter.protocol.http.gui.action;
+
+import java.awt.event.ActionEvent;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.swing.JFileChooser;
+
+import org.apache.jmeter.exceptions.IllegalUserActionException;
+import org.apache.jmeter.gui.action.AbstractActionWithNoRunningTest;
+import org.apache.jmeter.gui.action.ActionNames;
+import org.apache.jmeter.gui.util.FileDialoger;
+import org.apache.jmeter.protocol.http.correlation.Correlation;
+import org.apache.jmeter.protocol.http.gui.CorrelationGui;
+import org.apache.jmeter.util.JMeterUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CorrelationJMXFile extends AbstractActionWithNoRunningTest {
```

Comment:
Missing javadoc and name might not be adequate, give it a more clear name

Reply:
Updated the name and javadoc.

Reply:
A clear and adequate name has been updated. The name is changed from CorrelationJMXFile to "CorrelateJMXFile"
Also, the javadoc has updated to describe about the class.